_data/MadeleineGil/MadeleineGil.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Maddy
+institution: UTS 🚩
```

Comment:
I'm not sure, but I think the flag icon will be troublesome because it's a non-ascii char. I think it would be best to remove it to not cause any problems in the merge.

Reply:
have removed it just incase